examples/gallery/images/rgb_image.py

```diff
@@ -0,0 +1,37 @@
+"""
+RGB Image
+---------
+The :meth:`pygmt.Figure.grdimage` method can be used to plot Red, Green, Blue
+(RGB) images, or any 3-band false color combination. Here, we'll use
+:meth:`rioxarray.open_rasterio` to read a GeoTIFF file into an
+:class:`xarray.DataArray` format, and plot it on a map.
+
+The example below shows a Worldview 2 satellite image over
+`Lāhainā, Hawai'i during the August 2023 wildfires
+<https://en.wikipedia.org/wiki/2023_Hawaii_wildfires#L%C4%81hain%C4%81>`_.
+Data is sourced from a Cloud-Optimized GeoTIFF file hosted on
+`OpenAerialMap <https://map.openaerialmap.org>`_ under a
+`CC BY-NC 4.0 <https://creativecommons.org/licenses/by-nc/4.0/>`_ license.
+"""
+import pygmt
+import rioxarray
+
+###############################################################################
+# Read 3-band data from GeoTIFF into an xarray.DataArray object
+image = rioxarray.open_rasterio(
+    filename="https://oin-hotosm.s3.us-east-1.amazonaws.com/64d6a49a19cb3a000147a65b/0/64d6a49a19cb3a000147a65c.tif",
+    overview_level=5,
+)
+# Subset to area of Lāhainā in EPSG:32604 coordinates
+image = image.rio.clip_box(minx=738000, maxx=755000, miny=2300000, maxy=2318000)
+image
+
+###############################################################################
+# Plot the RGB imagery
+fig = pygmt.Figure()
+fig.grdimage(
+    grid=image,
+    projection="x1:100000",
```

Comment:
I feel we should add a comment explaining this projection argument.

----------

I just realized that the documentations of the Cartesian [linear](https://www.pygmt.org/dev/projections/nongeo/cartesian_linear.html), [logarithmic](https://www.pygmt.org/dev/projections/nongeo/cartesian_logarithmic.html), and [power](https://www.pygmt.org/dev/projections/nongeo/cartesian_power.html) projections only contain and show the _width_[/_hight_] option (further there is a typo regarding the `/` being outside the brackets) via an upper-case letter (**X**), but not the _scale_ option via a lower-case letter (**x**).

Reply:
Ok, added a comment in 9085f6586140a616d81ded1ff4050bed6377c805.

> projection="x1:100000",  # map scale where 1 cm on map equals 1 km on the ground